src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/AbsentReason.kt

```diff
@@ -1,15 +1,15 @@
 package uk.gov.justice.digital.hmpps.whereabouts.model
 
-enum class AbsentReason(val label: String) {
-  ApprovedCourse("Approved course"),
-  AcceptableAbsence("Acceptable absence"),
-  SessionCancelled("Session cancelled"),
-  RestInCellOrSick("Rest in cell or sick"),
-  RestDay("Rest day"),
-  UnacceptableAbsence("Unacceptable absence"),
-  NotRequired("Not required to attend"),
-  Refused("Refused to attend"),
-  RefusedIncentiveLevelWarning("Refused to attend");
+enum class AbsentReason(val label: String, val eventOutcome: String) {
+  ApprovedCourse("Approved course", "ACCAB"),
+  AcceptableAbsence("Acceptable absence", "ACCAB"),
+  SessionCancelled("Session cancelled", "CANC"),
+  RestInCellOrSick("Rest in cell or sick", "REST"),
+  RestDay("Rest day", "REST"),
+  UnacceptableAbsenceIncentiveLevelWarning("Unacceptable absence", "UNACAB"),
```

Comment:
we're going to add a new `UnacceptableAbsence` that doesn't have a warning, so rename this one prior to adding the new one.